src/registry/addressNL/edit.stories.ts

```diff
@@ -0,0 +1,160 @@
+import {Meta, StoryObj} from '@storybook/react';
+import {expect, userEvent, waitFor, within} from '@storybook/test';
+
+import ComponentEditForm from '@/components/ComponentEditForm';
+
+export default {
+  title: 'Builder components/AddressNL',
+  component: ComponentEditForm,
+  parameters: {},
+  args: {
+    isNew: true,
+    component: {
+      id: 'wekruya',
+      type: 'addressNL',
+      key: 'address',
+      label: 'An address field',
+    },
+    builderInfo: {
+      title: 'Address NL',
+      icon: 'home',
+      group: 'basic',
+      weight: 10,
+      schema: {},
+    },
+  },
+} as Meta<typeof ComponentEditForm>;
+
+type Story = StoryObj<typeof ComponentEditForm>;
+
+export const PostcodeValidationTabWithoutConfiguration: Story = {
+  name: 'AddressNL postcode validation tab (no prior configuration)',
+  play: async ({canvasElement, step}) => {
+    const canvas = within(canvasElement);
+
+    await step('Navigate to validation tab and open Postcode validation', async () => {
+      await userEvent.click(canvas.getByRole('link', {name: 'Validation'}));
+      await userEvent.click(canvas.getAllByText('Postcode')[0]);
+      await waitFor(async () => {
+        expect(await canvas.findByText('Regular expression for postcode')).toBeVisible();
+        expect(await canvas.findByText('NL')).toBeVisible();
+        expect(await canvas.findByText('EN')).toBeVisible();
+        expect(await canvas.findByText('Error code')).toBeVisible();
+        expect(await canvas.findByDisplayValue('pattern')).toBeVisible();
```

Comment:
I thought I set up accessible `aria-label` attributes in these components, so you should be able to `await canvas.getByLabelText(...)` these and then assert `expect(...).toHaveDisplayValue(...)` them.

Reply:
I'm not sure what you mean, would I be doing `canvas.getByLabelText("Error code")`? That doesn't seem to work because it doesn't have an `aria-label` (from what I can see)